src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -71,9 +72,7 @@ export const View = (props: Props) => {
     return new Date(isoString);
   };
 
-  const breachesDataArray = props.userBreaches
-    .map((elem: SubscriberBreach) => elem)
```

Comment:
@codemist I also removed this `.map`, as it didn't appear to do anything? But let me know if that was wrong.